airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -19,22 +19,28 @@
 Example Airflow DAG that shows how to use DisplayVideo.
 """
 import os
+from typing import Dict
 
 from airflow import models
+from airflow.providers.google.cloud.operators.gcs_to_bigquery import GCSToBigQueryOperator
 from airflow.providers.google.marketing_platform.operators.display_video import (
     GoogleDisplayVideo360CreateReportOperator, GoogleDisplayVideo360DeleteReportOperator,
     GoogleDisplayVideo360DownloadReportOperator, GoogleDisplayVideo360RunReportOperator,
-    GoogleDisplayVideo360UploadLineItemsOperator,
+    GoogleDisplayVideo360SDFtoGCSOperator, GoogleDisplayVideo360UploadLineItemsOperator,
 )
 from airflow.providers.google.marketing_platform.sensors.display_video import (
     GoogleDisplayVideo360ReportSensor,
 )
 from airflow.utils import dates
 
 # [START howto_display_video_env_variables]
-
 BUCKET = os.environ.get("GMP_DISPLAY_VIDEO_BUCKET", "gs://test-display-video-bucket")
-OBJECT_NAME = os.environ.get("OBJECT_NAME", "file.csv")
+PATH_TO_UPLOAD_FILE = os.environ.get("GCP_GCS_PATH_TO_UPLOAD_FILE", "test-gcs-example.txt")
+PATH_TO_SAVED_FILE = os.environ.get("GCP_GCS_PATH_TO_SAVED_FILE", "test-gcs-example-download.txt")
+BUCKET_FILE_LOCATION = PATH_TO_UPLOAD_FILE.rpartition("/")[-1]
+ADVERTISER_ID = os.environ.get("GMP_ADVERTISER_ID", 1234567891234567)
+SDF_VERSION = os.environ.get("GMP_SDF_VERSION", "SDF_VERSION_4_2")
```

Comment:
SDF version 4.2 has been deprecated.  Suggest switching to the latest version 5.1:
https://developers.google.com/bid-manager/guides/structured-data-file/v5-1/LineItem

Reply:
Done